### DISCUSSION TITLE
Simulate water and wind

### DISCUSSION THREAD

**Discussion by MartinDelille** [pmndrs/cannon-es#212] [General]
I started working on a personal project where I want to simulate a sailing boat: https://martin.delille.org/nautilus/

The source code lives here: https://github.com/MartinDelille/nautilus/ (and the deployed code lives in the waves branch for now.

For now the waves are only a threejs mesh and the boat fall on a plane body but I'm starting implementing buyancy.

I may document what I am doing here and would appreciate help and suggestion!

**Comment by dirkk0** [comment]
Maybe you can use something from here:
https://glitch.com/edit/#!/dk-buoyancy

**Reply by MartinDelille** [reply]
Very nice! Thanks for sharing! I will have a look!
